src/Zynga/Framework/IO/Disk/V1/Manager.hh

```diff
@@ -0,0 +1,136 @@
+<?hh // strict
+
+namespace Zynga\Framework\IO\Disk\V1;
+
+use Zynga\Framework\IO\Disk\V1\Exception\FailedToCloseFile as FailedToCloseFileException;
+use Zynga\Framework\IO\Disk\V1\Exception\FailedToCreateDirectory as FailedToCreateDirectoryException;
+use Zynga\Framework\IO\Disk\V1\Exception\FailedToOpenFile as FailedToOpenFileException;
+use Zynga\Framework\IO\Disk\V1\Exception\FailedToWriteToFile as FailedToWriteToFileException;
+use Zynga\Framework\IO\Disk\V1\Exception\ReadPermissions as ReadPermissionsException;
+use Zynga\Framework\IO\Disk\V1\Exception\WritePermissions as WritePermissionsException;
+
+/**
+ * Lightweight class for managing Disk IO
+ */
+class Manager {
```

Comment:
Optional:

If this implemented an interface and we have a static getter, we could override the singleton for testing purposes.  That would be useful for dependency injection later on for tests that use this class. Otherwise, we could just add a 'test' mode to this class but I prefer the initial approach.

Reply:
Additional ask:

Are there no tests for this change? is it missing as part of the PR?